src/nft/components/explore/Table.tsx

```diff
@@ -246,7 +247,8 @@ function LoadingTable({ headerGroups, visibleColumns, ...props }: LoadingTablePr
       <table {...props} className={styles.table}>
         <thead className={styles.thead}>
           {headerGroups.map((headerGroup) => (
-            <tr {...headerGroup.getHeaderGroupProps()} key={headerGroup.id}>
+            // eslint-disable-next-line react/jsx-key
+            <tr {...headerGroup.getHeaderGroupProps()}>
```

Comment:
the table row should be given a valid key instead of removing and disabling the error